dockers/docker-dhcp-server/cli-plugin-tests/test_show_dhcp_server.py

```diff
@@ -185,7 +185,9 @@ def test_show_dhcp_server_ipv4_option_without_name(self, mock_db):
 +---------------+-------------+-------------+--------+
 | Option Name   |   Option ID | Value       | Type   |
 +===============+=============+=============+========+
-| option60      |          60 | dummy_value | string |
+| option60      |         163 | dummy_value | string |
++---------------+-------------+-------------+--------+
+| option61      |         164 | dummy_value | string |
 +---------------+-------------+-------------+--------+
```

Comment:
Why option ID doesn't match option name?

Reply:
Option ID is used by dhcp server to mark a specific type of option, as defined in https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-dhcp-utilities/dhcp_utilities/dhcpservd/dhcp_option.csv , and there can be many entries with the same option id, but option name has to be unique